src/styles/mo-components/_social.scss

```diff
@@ -1,19 +1,71 @@
-.share_item, .share_item:active, .share_item:visited {
+.share_item {
   display: inline-flex;
   font-size: 1rem;
   line-height: 20px;
   padding: .5em;
   margin-right: 2em;
   color: $azure;
+
   svg {
     width: 20px;
     height: 20px;
     padding: 0;
     margin-right: 1em;
     fill: $azure;
   }
+
+  &:active,
+  &:visited {
+    color: $azure;
+
+    svg {
+      fill: $azure;
+    }
+  }
+
   &.flex {
     display: flex;
   }
-}
 
+  &__btn {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    @include btn--azure;
+    padding: 10px 20px;
```

Comment:
A little bottom margin would make these look less weird. Other than that, all looks good.

Reply:
Do we want to be prescriptive with margins in the component styles? I've been using the `mb-x` class on the containers. I added some for presentational purposes to the styleguide, but if you do want to add margins here I can do that too.